lib/stripe/connect/oauth.ex

```diff
@@ -175,7 +176,16 @@ defmodule Stripe.Connect.OAuth do
   """
   @spec authorize_url(map) :: String.t()
   def authorize_url(options \\ %{}) do
-    base_url = "https://connect.stripe.com/oauth/authorize?"
+    oAuthUrls = %{
+      "standard" => "https://connect.stripe.com/oauth/authorize?",
+      "express" => "https://connect.stripe.com/express/oauth/authorize?"
```

Comment:
Would you mind adding some documentation for this behaviour?

Reply:
Yes, would that be in readme or as comments inline with code?

Reply:
In the @doc attribute for this method.

Reply:
@andreyk-code Any updates?

Reply:
Sorry, I haven't had a chance to look at this again. Will try to soon. 

Any suggestions around getting the coveralls test to pass?

Reply:
That one is no biggie.  As long as it is testable and we can reasonably avoid a regression

Reply:
@andreyk-code Any updates?  Thx again for this PR!